src/convertToObject.js

```diff
@@ -3,16 +3,23 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  sourceString.split(';').forEach(stringProperty => {
+    const [prop, value] = stringProperty.split(':');
```

Comment:
I think key would be more exlicit
```suggestion
    const [key, value] = stringProperty.split(':');
```